README.md

```diff
@@ -19,9 +19,8 @@ Axom provides robust, flexible software infrastructure for the development of mu
 Documentation
 -------------
 
-Latest docs on Develop branch: https://axom.readthedocs.io
-
-To access docs for other versions: https://readthedocs.org/projects/axom/
+To access the latest docs for Axom as well as those for specific versions:
+https://app.readthedocs.org/projects/axom/
```

Comment:
I am not sure this is a helpful link for users.

https://axom.readthedocs.io/en/develop/

that is what i would expect to see. Otherwise they have to know to click the little book icon.

Reply:
I agree. I haven't figured out how to get to the latest version of develop docs because it appears that each build is versioned in the new desktop. I'll keep playing around with it.

Reply:
@white238 I twiddled with the dashboard settings. I think this is what we want now.